### ISSUE TITLE
Interstitial Closed Event?

### ISSUE THREAD

**Issue by RexfelisLOC** [AdsJumbo/AdsJumbo.com#1]
Would be nice to have a OnClosed event for the interstitial ad to resume anything paused while the ad is displayed.

**Comment by AdsJumbo** [comment]
Hi, Yes sure! We will definitely add this feature.

Otherwise you can reach us at mail: info@adsjumbo.com
If you want to activate your app/game just hit the button "Request Activation" this push all premium campaigns and demand.

Any feedback is welcome!

Regards,
AdsJumbo.com Team

**Comment by AdsJumbo** [comment]
Hi,

UPDATE: OnClosed event for the interstitial ad was added, Please update NuGet to version AdsJumbo - 5.5.5.

Just call:
<pre><code>InterstitialAd.OnAdClosed += InterstitialAd_OnAdClosed;</code></pre>

Handle:
<pre><code>private void InterstitialAd_OnAdClosed(object sender, RoutedEventArgs e)
{
  // When an Interstitial Ad is closed, just resume app / game according to your requirements
}</code></pre>
Otherwise you can reach us at mail: info@adsjumbo.com
If you want to activate your app/game just hit the button "Request Activation" this push all premium campaigns and demand.

Any feedback is welcome!

Regards,
AdsJumbo.com Team

**Comment by Linder-Boss** [comment]
Hi,

It seams OnAdClosed is being triggered twice. Can you confirm?

Thanks!

**Comment by AdsJumbo** [comment]
Hi,

OnAdClosed is triggered only once, per click to the close button (on interstitial ad).

Regards,
AdsJumbo.com Team
